README.org

```diff
@@ -60,31 +74,31 @@ t)
 
   As with ~#+context_header_extra~, snippets can override existing command definitions.
 
-- ~#+context_preset~ :: This keyword allows a new document "Preset" to be
-  defined. A "Preset" is a collection of some literal ConTeXt code to
-  insert into the preamble, a list of snippets, and an "inner template"
-  that specifies overall document structure.
+- ~#+context_preset~ :: This keyword to define a new document /Preset/.
+  A /Preset/ is a collection of some literal ConTeXt code to
+  insert into the preamble, a list of snippets,
+  and an "inner template" that specifies overall document structure.
```

Comment:
I noticed I wrote this wrong the first time. It doesn't _define_ the preset, it _selects_ it!

How about 
```org
~#+context_preset~ :: This keyword selects the document /Preset/.
  A /Preset/ is a collection of some literal ConTeXt code to
  insert into the preamble, a list of snippets,
  and an "inner template" that specifies overall document structure.
```